src/Microsoft.DotNet.Build.Tasks/GetPackageVersion.cs

```diff
@@ -34,7 +34,7 @@ public override bool Execute()
                 select el).FirstOrDefault();
 
             Debug.Assert(embeddedVerNode != null, "embeddedVerNode shouldn't be null");
-            PackageVersion = string.Format("{0}-{1}", embeddedVerNode.Value, DailyBuildNumber);
+            PackageVersion = string.Format("{0}-{1}", embeddedVerNode.Value, BuildNumber);
```

Comment:
I just tried it, still fails.  According to the SemVer spec the value must be alphanumeric.